src/components/SummaryWeather.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useWeather } from './WeatherProvider';
+import {
+  toCapitalized,
+  toLocalDate,
+  getHourString,
+  getIconSrc,
+} from '../utilities/helperFunctions';
+
+export default function SummaryWeather() {
+  const { currentWeather, hourlyWeather } = useWeather();
+  if (!(currentWeather && hourlyWeather)) return null;
+
+  const { timezone } = currentWeather;
+  const summaryHeader = `${currentWeather.name}, ${currentWeather.sys.country}`;
+  const summaryIconURL = getIconSrc(currentWeather.weather[0].icon);
+  const summaryIconAltText = `An icon representing ${currentWeather.weather[0].description}`;
+  const summaryDescription = `${toCapitalized(
+    currentWeather.weather[0].description
+  )}`;
+  const summaryTemperature = `${Math.round(currentWeather.main.temp)}°`;
+
+  const hourlyForecast = hourlyWeather.list
+    .slice(0, 4)
+    .map((weatherForHour) => (
+      <div key={weatherForHour.dt} className="row align-items-center">
+        <div className="col" />
+        <div className="col fs-4">
+          {getHourString(toLocalDate(weatherForHour.dt), timezone)}
+        </div>
+        <div className="col">
+          <img
+            src={getIconSrc(weatherForHour.weather[0].icon)}
+            alt={`An icon representing ${weatherForHour.weather[0].description}`}
+          />
+        </div>
+        <div className="col" />
+      </div>
+    ));
+
+  return (
+    <div className="summary-weather container py-4 text-center border border-primary rounded">
+      <h2 className="row justify-content-center fs-1">{summaryHeader}</h2>
```

Comment:
Adding the bootstrap class font-size (fs) seems unnecessary because the content is wrapped in a header tag. Essentially, it sorta contradicts each other because you use h2 to decrease the font-size from an h1, but then use fs-1 to increase the font size. I think for readability to ensuring that the tag and bootstrap style doesn't cause confusion, either use a header tag without the fs, use the h2 bootrap class, or use fs but with a p tag instead. https://getbootstrap.com/docs/5.0/utilities/text/